recipes/doc2dash/meta.yaml

```diff
@@ -0,0 +1,75 @@
+{% set name = "doc2dash" %}
+{% set version = "2.2.0" %}
+{% set hash_type = "sha256" %}
+{% set hash_value = "09cbd5ca57848e9ba40c25c693a1801e11b6aeb87770f5c764d4ad1742a7ea14" %}
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  fn: '{{ name }}-{{ version }}.tar.gz'
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  '{{ hash_type }}': '{{ hash_value }}'
+
+build:
+  noarch: python
+  number: 0
+  entry_points:
+    - doc2dash = doc2dash.__main__:main
+  script: python setup.py install  --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
```

Comment:
The `--single-version-externally-managed` flag to `setup.py` means that you only need packages in the list `build` requirements that are either imported by `setup.py` or are in `setup_requires`. I think you only need `python` and `setuptools` here.

Reply:
Ok, trimmed down the build requirements in https://github.com/conda-forge/staged-recipes/pull/4121/commits/cf80346fd2a6bcaf4e18e8b2b84c31d6842c2b13 . I confirmed that build succeeds locally after this change.